docs/pydoc/config/rankers_api.yml

```diff
@@ -1,8 +1,7 @@
 loaders:
   - type: haystack_pydoc_tools.loaders.CustomPythonLoader
     search_path: [../../../haystack/components/rankers]
-    modules: ["lost_in_the_middle", "meta_field", "meta_field_grouping_ranker", "transformers_similarity",
-              "sentence_transformers_diversity", "sentence_transformers_similarity"]
+    modules: ["lost_in_the_middle", "meta_field", "meta_field_grouping_ranker", "transformers_similarity", "sentence_transformers_diversity"]
```

Comment:
please add the missing modules and especially reintroduce `sentence_transformers_similarity`